src/main/java/uk/gov/companieshouse/overseasentitiesapi/service/OverseasEntitiesService.java

```diff
@@ -232,9 +235,9 @@ private void updateTransactionWithLinksAndCompanyName(Transaction transaction,
     public Optional<OverseasEntitySubmissionDto> getOverseasEntitySubmission(String submissionId) {
         var submission = overseasEntitySubmissionsRepository.findById(submissionId);
         if (submission.isPresent()) {
-            ApiLogger.info(String.format("%s: Overseas Entities Submission found. About to return", submission.get().getId()));
-
-            var dto = overseasEntityDtoDaoMapper.daoToDto(submission.get());
+            var overseasEntitySubmissionDao = submission.get();
+            ApiLogger.info(String.format("%s: Overseas Entities Submission found. Version %s. About to return", overseasEntitySubmissionDao.getId(), overseasEntitySubmissionDao.getSchemaVersion()));
```

Comment:
Minor, but would output 'Schema version' in the log message.

Reply:
Updated.